src/Sentry/Internal/ActiveProcessInfo.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Diagnostics;
+
+namespace Sentry.Internal
+{
+    internal sealed class ActiveProcessInfo : IActiveProcessInfo
+    {
+        public static IActiveProcessInfo Instance { get; } = new ActiveProcessInfo();
+
+        public int GetCurrentProcessId()
+        {
+            using var process = Process.GetCurrentProcess();
```

Comment:
shall we use prep for the new net6 process apis? https://devblogs.microsoft.com/dotnet/new-dotnet-6-apis-driven-by-the-developer-community/#process-path-id
```
#if NET6_0_OR_GREATER
            return Environment.ProcessId
#else
            using var process = Process.GetCurrentProcess();
            return process.Id;
#endif
```

Reply:
Seems like this is .NET 5 even: https://docs.microsoft.com/en-us/dotnet/api/system.environment.processid?view=net-5.0